### DISCUSSION TITLE
Is there anyway to persist the message until interaction, or at least keep it posted for longer?

### DISCUSSION THREAD

**Discussion by DanPrs** [Windos/BurntToast#97] [General]
The message disappears rather quickly. If a user happens to look away, they might miss it entirely.

**Comment by Windos** [comment]
Hey @DanPrs, unfortunately there's no way to hold a toast on screen until it is interacted with. It will end up going to the Action Center for later action by the user (though if they're anything like me they'll just click the dismiss all button 😛).

We can however make the toast stay onscreen longer. Behind the scenes they use a concept call scenarios, the default one if I remember correctly is a "reminder" which is just a "ding, something happened, here's some info".

You can make use of an "alarm"  scenario which shows the toast for much longer... and also has a more persistent sound to them.

The easiest way to do this is by choosing any of the Alarm or Call sounds on the `New-BurntToastNotification` function:

```PowerShell
 New-BurntToastNotification -Sound Alarm5
```

This shows on screen for 25 seconds.

If you want to really crank this up, you have to dive into the "advanced functions":

```powershell
$Text1 = New-BTText -Content 'Default Notification'
$ImagePath = Join-Path -Path (Get-Module BurntToast -ListAvailable).ModuleBase -ChildPath 'Images\BurntToast.png'
$Image1 = New-BTImage -Source $ImagePath -AppLogoOverride -Crop Circle
$Audio1 = New-BTAudio -Source 'ms-winsoundevent:Notification.Looping.Call9' -Loop

$Binding1 = New-BTBinding -Children $Text1 -AppLogoOverride $Image1
$Visual1 = New-BTVisual -BindingGeneric $Binding1
$Content1 = New-BTContent -Visual $Visual1 -Audio $Audio1 -Duration Long -Scenario IncomingCall

Submit-BTNotification -Content $Content1
```

The key here if on `New-BTContent` setting the duration to long and the scenario to incoming call. This will stay on the screen for about **5 minutes**.

You'll note that with `New-BTAudio` you have to use one of the looping (call or alarm) sounds, otherwise it defaults to the first call option, however you can remove the `-Loop` switch and it will only play the sound once and then stay on the screen in silence.

**Comment by MathFlyAIBV** [comment]
Hey,

Found out that if you change the registry key value of the key called MessageDuration located in HK_Current_User\Control Panel\Accessibility to a decimal of your choice, it will change the notification timing to whatever second you set as decimal.
I set 60 as the decimal value (3c in Hex) in my env and the notifications stays for 60 seconds on the screen. By default, It's set to 5.

Of Course, all notifications will be affected.

Cheers,

**Comment by ITistheworst** [comment]
I have found that `-Scenario Reminder` produces a notification that stays on screen until the user interacts with it. I have been using with `-Duration Long`, but I am not sure if that is necessary. E.g:

```powershell
$content = New-BTContent -Visual $visual -Duration Long -Scenario Reminder
```

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
